delfin/drivers/utils/ssh_client.py

```diff
@@ -278,3 +280,44 @@ def do_exec(self, command_str):
                 'is not a recognized command' in result:
             raise exception.StorageBackendException(result)
         return result
+
+    def do_exec_command(self, command_list):
+        result = ''
+        try:
+            with self.item() as ssh:
+                if command_list is not None and len(command_list) > 0 \
+                        and ssh is not None:
+                    channel = ssh.invoke_shell()
+                    for command in command_list:
+                        utils.check_ssh_injection(command)
+                        channel.send(command + '\n')
+                        time.sleep(0.5)
+                    channel.send("exit" + "\n")
+                    channel.close()
+                    while True:
+                        resp = channel.recv(9999).decode('utf8')
+                        if not resp:
+                            break
+                        result += resp
+        except paramiko.AuthenticationException as ae:
```

Comment:
There are too many duplicate codes in `do_exec` and `do_exec_command`, these two functions should be optimized.